src/DynamoCoreWpf/UI/Prompts/FunctionNamePrompt.xaml

```diff
@@ -1,57 +1,383 @@
-            <TextBlock Name="nameView" Foreground="WhiteSmoke" FontSize="13" Padding="5"></TextBlock>
-
-            <Label Content="{x:Static p:Resources.CustomNodePropertyWindowDescription}" Foreground="DarkGray" Height="28"  HorizontalAlignment="Left" Name="DescriptionLabel" VerticalAlignment="Top" >
-                <Label.ToolTip>
-                    <TextBlock Text="{x:Static p:Resources.CustomNodePromptDescriptionTooltip}" />
-                </Label.ToolTip>
-            </Label>
-            <TextBox Style="{DynamicResource SDarkTextBox}" Height="64" HorizontalScrollBarVisibility="Disabled" VerticalScrollBarVisibility="Auto" 
-                     Name="DescriptionInput" Padding="5" MinLines="3" TextWrapping="Wrap" AcceptsReturn="True" Tag="{x:Static p:Resources.CustomNodePropertyWindowDescriptionHint}"/>
-
-            <Label Content="{x:Static p:Resources.CustomNodePropertyWindowCategory}" Foreground="DarkGray" Height="28" HorizontalAlignment="Stretch" Name="label2"  />
-            <ComboBox FontSize="14" Style="{DynamicResource ResourceKey=SComboBox}" Background="Black" Foreground="White" HorizontalAlignment="Stretch" Name="categoryBox" Margin="0" IsEditable="True" TabIndex="1"/>
-
-            <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="0,25,0,0">
-
-                <Button Content="{x:Static p:Resources.OKButton}" Style="{DynamicResource ResourceKey=STextButton}" Margin="5" HorizontalAlignment="Right" Name="okButton" VerticalAlignment="Top" Width="auto" MinWidth="75" Click="OK_Click" IsDefault="True" TabIndex="2" />
-                <Button Content="{x:Static p:Resources.CancelButton}" Style="{DynamicResource ResourceKey=STextButton}" Margin="5" HorizontalAlignment="Right" Name="cancelButton" VerticalAlignment="Top" Width="auto" MinWidth="75" Click="Cancel_Click" IsCancel="True" TabIndex="3" />
-
-            </StackPanel>
-
-        </StackPanel>
+    <Grid Name="mainGrid">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="58px" />
+            <RowDefinition Height="400" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <!--  Top Section: Title + Close Button  -->
+        <Border Name="titleBar"
+                Grid.Row="0"
+                Grid.RowSpan="3"
+                Background="{StaticResource DarkMidGreyBrush}"
+                CornerRadius="4"
+                MouseDown="FunctionNamePrompt_MouseDown">
+            <DockPanel HorizontalAlignment="Stretch">
+
+                <!--  Close Button  -->
+                <Button x:Name="closeButton"
+                        Margin="0,20,25,20"
+                        VerticalAlignment="Top"
+                        Click="Cancel_Click"
+                        DockPanel.Dock="Right"
+                        KeyboardNavigation.IsTabStop="False"
+                        Style="{DynamicResource CloseButtonStyle}" />
```

Comment:
Does this close button come with blue hover over style?

Reply:
Yes, this is the common x button style:

![iQsXpFf2K7](https://user-images.githubusercontent.com/29973601/142451028-05692cf5-0680-4766-be56-419778efe9e6.gif)